docs/upgrade/UPGRADE-7.0.0-beta2.md

```diff
@@ -1,5 +1,6 @@
-## [From 7.0.0-beta1 to 7.0.0-beta2](https://github.com/shopsys/shopsys/compare/v7.0.0-beta1...v7.0.0-beta2)
-### [shopsys/project-base]
+# [From 7.0.0-beta1 to 7.0.0-beta2](https://github.com/shopsys/shopsys/compare/v7.0.0-beta1...v7.0.0-beta2)
```

Comment:
Now I really feel like a nitpick, however, you missed the word "Upgrade" in the beginning here

Reply:
No, it's perfectly ok :) I have to be more careful.